src/core/blockchain/types/client.ts

```diff
@@ -15,6 +16,7 @@ export abstract class BlockchainGenericClient {
     public nameService: IBlockchainNameService;
     public http: HttpClient;
     public readonly chainId: ChainIdType;
+    public clientUtils: IClientUtils;
```

Comment:
the property name should be `utils` it would be more simple when using it `getClient(...).utils....`